WooCommerce/Classes/Notifications/LocalNotification.swift

```diff
@@ -19,6 +19,8 @@ struct LocalNotification {
         case loginSiteAddressError = "site_address_error"
         case invalidEmailFromSiteAddressLogin = "site_address_email_error"
         case invalidEmailFromWPComLogin = "wpcom_email_error"
+        case invalidPasswordFromSiteAddressLogin = "site_address_password_error"
```

Comment:
❓ I didn't follow the tracking plan, but do we want to be more specific here, i.e with `site_address_wpcom_password_error`? Just in case we want to track for the wp-admin password later?

Reply:
Great catch on another password error scenario! updated in https://github.com/woocommerce/woocommerce-ios/pull/7422/commits/2b1689be0bafec15e234bbf831833bafccefbe87, and synced with Android in pe5sF9-h9-p2#comment-607